client/reader/search-stream/suggestion-provider.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import { map, sampleSize } from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import TagSubscriptions from 'lib/reader-tags/subscriptions';
+import i18nUtils from 'lib/i18n-utils';
+import { suggestions } from 'reader/search-stream/suggestions';
+
+export default ( Element, count = 3 ) => {
+	return class SuggestionsListener extends React.Component {
+
+		state = {
+			suggestions: this.getSuggestions()
+		}
+
+		componentWillMount() {
+			TagSubscriptions.on( 'change', this.handleChange );
+		}
+
+		componentWillUnmount() {
+			TagSubscriptions.off( 'change', this.handleChange );
+		}
+
+		suggestionsFromTags() {
```

Comment:
good call. how many tags do you think we should require before using them as a source?

or maybe we should mix tags and the hard-wired suggestions?